controllers/comment.js

```diff
@@ -152,4 +163,46 @@ export default class CommentController {
       });
     }
   }
+
+  /**
+* @description - Like/unlike an article comment
+* @static
+*
+* @param {object} req - HTTP Request
+* @param {object} res - HTTP Response
+*
+* @memberof CommentController
+*
+* @returns {string} Like/unlike comment result
+*/
+  static async likeComment(req, res) {
+    const { id } = req.params;
+    const { id: userId } = req.user;
+    try {
+      // Delete comment like, if already liked by user
+      if (await isCommentLikedByUser(id, userId)) {
```

Comment:
@olajide1234 there might be a Sequelize method to achieve this.

Reply:
I have updated the code to use a Sequelize method.